test/test-coordinator.js

```diff
@@ -156,6 +156,11 @@ TestCoordinator.prototype.start = function start(callback) {
     var self = this;
     callback = callback || function(){};
 
+    self.once('joined', function joined(){
+        // Set firstJoinDate to current Date if not already set.
+        self.firstJoinDate = self.firstJoinDate || new Date();
```

Comment:
I think keeping track of the last join date is easy and desirable as well. iirc most of the ringpop logics start after the last join.